fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -46,32 +46,36 @@ def __init__(
         self.pinned = False
         self.completed_at = None
         self.failed_at = None
+        self.scheduled_at = None
         self.result = None
         self.id = None
         self._doc = None
         self.metadata = None
 
     def from_pymongo(self, doc: dict):
         # required fields
-        self.operator = doc["operator"]
-        self.queued_at = doc["queued_at"]
-        self.run_state = doc["run_state"]
-        self.label = doc["label"] if "label" in doc else None
-        self.updated_at = doc["updated_at"] if "updated_at" in doc else None
+        self.operator = doc.get("operator")
+        self.queued_at = doc.get("queued_at")
+        self.run_state = doc.get("run_state")
```

Comment:
Don't really need to use `get()` for these as they're required to be set. It just pushes the errors to later